Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/bugreporting/event/BugEvent.kt

```diff
@@ -0,0 +1,20 @@
+package de.rki.coronawarnapp.bugreporting.event
+
+import org.joda.time.Instant
+
+interface BugEvent {
+    val id: Long
```

Comment:
My thinking was, that storing a long is more efficient/performant as storing a UUID string, because string is object and more complex that long. As I would have to write a little converter for it and also ordering uuids is difficult I really would like to stay on long-ID. But I can change it if you insist.

Reply:
I prefer the UUID, too. But would also be fine for me to have a follow-up ticket for this.

Reply:
For the amount of data we regenerate, the efficiency aspect does not matter.

The data in the database can have a `Long` ID, that does not need to be changed, but that's a detail of the storage class and should not be part of the core `BugEvent` class, i.e. should not be exposed to the other classes via the interface.

For easier bug finding and reference, let's give the core `BugEvent` class a `bugId: UUID` property, that a bug can then be independently of how it is stored, be referenced and found by.

Reply:
all right. adjusted.